acos_client/client.py

```diff
@@ -93,16 +93,20 @@
 class Client(object):
 
     def __init__(self, host, version, username, password, port=None,
-                 protocol="https", timeout=None, retry_errno_list=None):
+                 protocol="https", max_retries=3, timeout=5, retry_errno_list=None):
         self._version = self._just_digits(version)
         if self._version not in acos_client.AXAPI_VERSIONS:
             raise acos_errors.ACOSUnsupportedVersion()
+        self.max_retries = max_retries  # number of attempts to connect before giving up.
+        self.timeout = timeout  # give up after waiting this long for any data from remote device.
         self.host = host
         self.port = port
         self.http = VERSION_IMPORTS[self._version]['http'].HttpClient(
-            host, port, protocol, timeout=timeout, retry_errno_list=retry_errno_list)
+            host, port, protocol, timeout=timeout, max_retries=self.max_retries, retry_errno_list=retry_errno_list
```

Comment:
In the `__init__` we have `max_retries` first and `timeout` second but then here it is reversed.  Would be nice if it was consistent.